converters/fromZigbee.js

```diff
@@ -1699,7 +1699,7 @@ const converters = {
                 result.hysterersis = precisionRound(data[0x100A], 2) / 10;
             }
             if (data.hasOwnProperty(0x100B)) { // DisplayAutoOffEnable
-                const lookup = {0: 'enable', 1: 'disable'};
+                const lookup = {0: 'enable', 1: 'disabled'};
```

Comment:
to be consistent; should `enable` be renamed to `enabled`?

Reply:
Yes, I agree. Will fix after weekend